Fixed the bug in social media icons + Added instructions to customize the template in readme.md

ishubham21 opened the pull request.

ishubham21 committed:
fixing social media icons not working + adding instructions to customize the template in Readm.md

ishubham21 committed:
Merge pull request #5 from amin-codes/fix-socials

fixing social media icons not working + adding instructions to customize the template in Readme.md

grace-omotoso reviewed (changes requested):
@ishubham21 and other teammates. Can you make this work even without the user making changes to the code e.g change 
- github.com/username to https://github.com/MLH-Fellowship/
- email to fellowship@mlh.io
- Linkedin: https://www.linkedin.com/
- Twitter: https://twitter.com
It is a better user experience when the link works and it takes the user to a social media site even if it does not take them to their profile. It is better than it not working. For example, https://linkedin.com works and https://linkedin.com/username does not work.

Also, remove all commented code before sending a PR

amin-codes committed:
change links

amin-codes committed:
minor change to all.scss

ishubham21 committed:
fixed dark-mode in pod member section

ishubham21 committed:
fixed overflow in mobile view + centered the name of pod and submit button in contact form + fixed styling of about-us section

amin-codes committed:
changess

amin-codes committed:
Merge branch 'main' of github.com:amin-codes/pod-3.1.0-portfolio into main

ishubham21 committed:
Merge branch 'main' into responsiveness

ishubham21 committed:
Merge pull request #7 from amin-codes/responsiveness

Responsiveness + Dark Mode for pod members section

ishubham21 committed:
Update README.md

ishubham21 commented:
For issue#27, we have also done the following things -
1. Fixed responsiveness in mobile view
2. Centered pod title
3. Centered submit button in the contact form
4. Added dark mode for pod member cards
5. Removed some commented code
